ci-operator/step-registry/sandboxed-containers-operator/create-prowjob/sandboxed-containers-operator-create-prowjob-commands.sh

```diff
@@ -190,7 +190,7 @@ if [[ "${ENABLE_MUST_GATHER}" != "true" && "${ENABLE_MUST_GATHER}" != "false" ]]
 fi
 
 # Must-gather image to use for collecting debug information
-MUST_GATHER_IMAGE="${MUST_GATHER_IMAGE:-quay.io/openshift/origin-must-gather:latest}"
+MUST_GATHER_IMAGE="${MUST_GATHER_IMAGE:-registry.redhat.io/openshift-sandboxed-containers/osc-must-gather-rhel9:latest}"
```

Comment:
Oh I see, you put these into a separate commit. It's more common to force-push the changes, which is easier to review (one can click on "diff" to see what changed but still review the overall resulting commits, which is what is going to be merged). Note unless necessary it's good to keep the old HEAD while rebasing to allow the overall diff...